<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="ruleEditor">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="ruleNameViewer">
        <item>
         <widget class="QLabel" name="ruleNameLabel">
          <property name="text">
           <string>Rule name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ruleNameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="sourceBtn">
           <property name="text">
            <string>Folder Source</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="folderSourceLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(192, 28, 40)</string>
           </property>
           <property name="text">
            <string>No Directory Chosen</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ifLabel">
        <property name="text">
         <string>If any of the following conditions are met:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ifViewer">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="ruleComboBox">
           <item>
            <property name="text">
             <string>File</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Extension</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="verbComboBox">
           <item>
            <property name="text">
             <string>contains</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>is</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>begins with</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="chooseFolderLayout">
        <item>
         <widget class="QLabel" name="thenLabel">
          <property name="text">
           <string>Do the following to the matched folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="destBtn">
          <property name="text">
           <string>Choose Folder</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="thenGroupBox">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="actionComboBox">
           <item>
            <property name="text">
             <string>Move</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="toFolderLabel">
           <property name="text">
            <string>to folder:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="destFolderLabel">
           <property name="text">
            <string>No Destination Folder Chosen</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="applyViewer"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="canceRuleBtn">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>canceRuleBtn</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>canceRuleBtn</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
